### PR TITLE
Fundamental documentation

### PR DESCRIPTION
**Pull request by agusbusta**
Swagger was updated with all fundamentals routes. 

Link: https://novatidelabs-company.monday.com/boards/1398916040/pulses/1705773898

### CODE REVIEW

**Comment by daviddflix** [comment]
# Feedback

## 1. Conflict in `swagger.json`

### Issue:
When attempting to merge or review the PR, I encountered a conflict in the file `static/swagger.json`. This needs to be resolved before proceeding with further review.

### Suggested Action:
Please resolve the conflict in `static/swagger.json`. You can use the web editor or your local environment to handle this conflict.

---

## 2. Swagger Documentation Structure Issue

### Problem:
In the current Swagger documentation, the new endpoints added are referencing a different section as their tag, which is causing an issue with how the documentation is organized.

- **Existing GET methods** are under a section called **"Fundamentals"**.
- The **new endpoints** (POST, PUT, DELETE) are being referenced under a new section (due to a different tag), while the GET method remains under **"Fundamentals"**.

This will result in inconsistent documentation where:
- GET methods stay in **"Fundamentals"**.
- POST, PUT, DELETE methods are placed in a separate section.

### Suggested Fix:
Please adjust the tags for the new endpoints so that all related methods (GET, POST, PUT, DELETE) are grouped under the same section. Ensure that they are properly categorized to avoid splitting them across multiple sections.

---

## 3. Next Steps:
- Resolve the conflict in `static/swagger.json`.
- Review and adjust the Swagger tags for consistency across all HTTP methods (GET, POST, PUT, DELETE).
- Once these changes are made, I will proceed with reviewing the PR again.

Let me know if you need any assistance with resolving these issues!

Thanks!

**Comment by agusbusta** [comment]
Done!

**Comment by daviddflix** [comment]
# PR Feedback

Hey @agusbusta, there’s a new migration file in the `develop` branch that isn’t in this branch, which is preventing the server from starting. Could you take a look to resolve the issue?

## Suggested Steps

1. Switch to the `develop` branch.
2. Pull the latest changes to update `develop`.
3. Switch back to this branch.
4. Merge `develop` into this branch.
5. Resolve any conflicts that arise.

---

### Important

There are new endpoints in the Swagger JSON. **Please ensure these are not overwritten.** You can refer to the current documentation to compare it with the final docs in this branch. The final Swagger documentation should include all existing endpoints as well as the new "Fundamentals" documentation.

### Documentation Structure

Please ensure the sections are organized as follows:

- **Fundamentals**
  - Introduction
  - Revenue Model
- **Tokenomics**
- **Competitors**
- **Hacks**
- **Upgrades**
- **DApps**

Thank you!

**Comment by agusbusta** [comment]
Fixed!

**Comment by daviddflix** [comment]
Hey @agusbusta you have created new sections for hacks, dapps, upgrades, tokenimics, competitor... then the methods in the following section should be their respective section.

image

**Comment by agusbusta** [comment]
Develop + fix/routes-swagger-fund branch, are now merged! so its updated to latest version!

**Comment by daviddflix** [comment]
Hey @agusbusta were the other request changes also applied?

File: all_boards_data.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (daviddflix) [inline all_boards_data.json:1]: Hey @agusbusta is this necessary?

File: monday_client.py
Comment (daviddflix) [inline monday_client.py:1]: @agusbusta this should not be here, right?

**Review by daviddflix — CHANGES_REQUESTED** [review]

**Review by daviddflix — APPROVED** [review]

**Pull request merged**